src/Model/State.re

```diff
@@ -125,6 +125,21 @@ let defaultKeyBindings =
         ~command=Commands.List.select.id,
         ~condition="listFocus" |> WhenExpr.parse,
       ),
+      bind(
+        ~key="<C-v>",
+        ~command=Commands.List.selectVertical.id,
+        ~condition="listFocus" |> WhenExpr.parse,
+      ),
```

Comment:
The only concern I have here is that the `Control+V` keybinding would conflict with paste on Windows/Linux (ie, regressing https://github.com/onivim/oni2/issues/2308)

A couple ideas:
- We could use `Control+Shift+V` for opening in vertical tab
- We could use the `Shift+CR` keybinding for vertical, and `Control+T` for new tab